docs/devcontainer-example/Dockerfile.llvm-project

```diff
@@ -0,0 +1,15 @@
+# By default, use ubuntu:focal
+ARG BASE_IMAGE="onnxmlirczar/onnx-mlir-llvm-static:amd64"
```

Comment:
Change the comment above or remove it.

The Docker.md probably should mention that things like Mac M1/2 will require virtualization such as Rosetta, which is still under development (https://github.com/docker/roadmap/issues/384), to run amd64 containers. Qemu can also work but will be pretty slow.